resolwe_bio/descriptors/reads_descriptorschema.yaml

```diff
@@ -0,0 +1,20 @@
+- model: flow.descriptorschema
+  pk: 1001
+  fields:
+    slug: reads
+    version: 0.0.0
+    name: reads_annotation
+    created: 2016-03-17 09:04:07.325661+00:00
+    modified: 2016-03-17 09:04:07.325937+00:00
```

Comment:
Check if importing this fixture yields a warning. If so, write the date as string.